### PR TITLE
400 translation check

### PR DESCRIPTION
**Pull request by wouter1975**
Translation updates in NL and EN
- added translation to a couple of component
- Some translation change of #359 already included.

### CODE REVIEW

File: apps/wfo-ui/pages/index.tsx [hunk 1]
```diff
@@ -11,10 +12,11 @@ import {
 
 export function Index() {
     const { data: session } = useSession();
+    const t = useTranslations(`main`);
```
Comment (DutchBen) [inline apps/wfo-ui/pages/index.tsx:15]: Is that a backtick or a single quote? Single quotes should be used here

File: packages/orchestrator-ui-components/src/components/WfoSettingsPage/WfoEngineStatusButton.tsx [hunk 2]
```diff
@@ -13,6 +15,7 @@ export const WfoEngineStatusButton: FC<WfoEngineStatusButtonProps> = ({
     engineStatus,
     changeEngineStatus,
 }) => {
+    const t = useTranslations(`settings.page`);
```
Comment (DutchBen) [inline packages/orchestrator-ui-components/src/components/WfoSettingsPage/WfoEngineStatusButton.tsx:18]: Single quotes or backticks?

File: packages/orchestrator-ui-components/src/components/WfoSettingsPage/WfoModifySettings.tsx [hunk 3]
```diff
@@ -13,21 +15,23 @@ export type WfoModifySettingsProps = {
...
-        </EuiText>
-        <EuiSpacer size="m"></EuiSpacer>
-        <WfoEngineStatusButton
-            engineStatus={engineStatus}
-            changeEngineStatus={changeEngineStatus}
-        />
-    </EuiPanel>
-);
+}) => {
+    const t = useTranslations(`settings.page`);
```
Comment (Georgi2704) [inline packages/orchestrator-ui-components/src/components/WfoSettingsPage/WfoModifySettings.tsx:19]: ^

File: apps/wfo-ui/pages/index.tsx (hunk 1 above)
Comment (wouter1975) [reply apps/wfo-ui/pages/index.tsx:15]: check

File: packages/orchestrator-ui-components/src/components/WfoSettingsPage/WfoEngineStatusButton.tsx (hunk 2 above)
Comment (wouter1975) [reply packages/orchestrator-ui-components/src/components/WfoSettingsPage/WfoEngineStatusButton.tsx:18]: check

File: packages/orchestrator-ui-components/src/components/WfoSettingsPage/WfoModifySettings.tsx (hunk 3 above)
Comment (wouter1975) [reply packages/orchestrator-ui-components/src/components/WfoSettingsPage/WfoModifySettings.tsx:19]: check

File: packages/orchestrator-ui-components/src/components/WfoSettingsPage/WfoStatus.tsx [hunk 4]
```diff
@@ -19,6 +21,7 @@ export type WfoStatusProps = {
 export const WfoStatus: FC<WfoStatusProps> = ({ engineStatus }) => {
     const { theme } = useOrchestratorTheme();
     const isRunning = engineStatus === 'RUNNING';
+    const t = useTranslations(`settings.page`);
```
[the remark this answers is not in the document]
Comment (wouter1975) [reply packages/orchestrator-ui-components/src/components/WfoSettingsPage/WfoStatus.tsx:24]: check

File: packages/orchestrator-ui-components/src/components/WfoSettingsPage/WfoFlushSettings.tsx [hunk 5]
```diff
@@ -23,6 +25,7 @@ const clearCache = async (apiUrl: string, settingName: string) => {
 
 export const WfoFlushSettings: FunctionComponent = () => {
     const { orchestratorApiBaseUrl } = useContext(OrchestratorConfigContext);
+    const t = useTranslations(`settings.page`);
```
[the remark this answers is not in the document]
Comment (wouter1975) [reply packages/orchestrator-ui-components/src/components/WfoSettingsPage/WfoFlushSettings.tsx:28]: check

File: packages/orchestrator-ui-components/src/components/WfoTable/WfoTableWithFilter/WfoTableWithFilter.tsx [hunk 6]
```diff
@@ -83,6 +85,7 @@ export const WfoTableWithFilter = <T,>({
     const [showSettingsModal, setShowSettingsModal] = useState(false);
     const [selectedDataForDetailModal, setSelectedDataForDetailModal] =
         useState<T | undefined>(undefined);
+    const t = useTranslations(`common`);
```
Comment (ricardovdheijden) [inline packages/orchestrator-ui-components/src/components/WfoTable/WfoTableWithFilter/WfoTableWithFilter.tsx:88]: Backtick

Comment (wouter1975) [reply packages/orchestrator-ui-components/src/components/WfoTable/WfoTableWithFilter/WfoTableWithFilter.tsx:88]: check

**Pull request merged**